contributors/design-proposals/encryption.md

```diff
@@ -0,0 +1,204 @@
+# Encryption
+
+## Abstract
+
+The scope of this proposal is to ensure that resources can be encrypted at the datastore layer. Encrypting data at REST and via third party vendors is a desired feature but outside the scope of this proposal. Encryption will be optional for any resource but we suspect it will be used for the Secret resource in most cases.
```

Comment:
Yes, just the implementation is outside the scope. When we develop the built in provider, it should be behind a pluggable interface. 

I think my understanding of "encryption at rest" may be wrong here. My understanding of "encryption at rest" is that HTTP calls between a client and the server will have secrets encrypted. This requires a lot more work since the `kubelet` now needs to have the encryption keys as well. For this proposal we meant that the secret stored in etcd will be encrypted. I'll make sure to clarify that.